dynamiq/storages/vector/pinecone/pinecone.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         connection: Pinecone | None = None,
         client: Optional["PineconeClient"] = None,
         index_name: str = "default",
+        content_key: str = "content",
```

Comment:
Please add new params at the end to avoid breaking changes as users could pass params without keys.